common/dests.go

```diff
@@ -66,4 +66,5 @@ const (
 	TingyunDestinationType                DestinationType = "tingyun"
 	TraceloopDestinationType              DestinationType = "traceloop"
 	UptraceDestinationType                DestinationType = "uptrace"
+	VictoriaMetricsDestinationType        DestinationType = "victoriametrics"
```

Comment:
This should also mention "cloud" so we don't have a conflict once we introduce the self-hosted victoriametrics destination